themes/bootstrap3/templates/search/history-table.phtml

```diff
@@ -3,25 +3,34 @@
   $saveSupported = $this->accountCapabilities()->getSavedSearchSetting() === 'enabled';
 ?>
 <table class="search-history-table" id="<?=$this->showSaved ? 'saved-searches' : 'recent-searches'?>">
+  <thead>  
```

Comment:
@stephanieleary, I see this is a draft, so maybe I'm reviewing prematurely... but there seem to be some doubled and oddly nested tags here -- e.g. double open theads, double close tbodys, etc. Can you take another look at that?

Reply:
What an utterly bizarre glitch! I checked my local file and I don't see the duplicated lines, so I'm not sure what happened. I've corrected the file.

Reply:
I can't explain it, but thanks for fixing it! :-)